src/condor_tests/test_xfer_hold_codes.py

```diff
@@ -123,6 +123,7 @@ def test_jobOutputFailureAP(self, jobOutputFailureAP):
 
    def test_jobInputFailureEP(self, jobInputFailureEP):
       assert jobInputFailureEP["HoldReasonCode"] == 13
+      assert jobInputFailureEP["HoldReasonSubCode"] == (1 << 8)
```

Comment:
The current bug is that HoldReasonSubCode is _always_ `(1 << 8)`, so this test is not just mostly useless, but also deceptive.  You'll probably have to write a plugin that exits with different codes depending on its URLs to make sure the error reporting is working properly.

Reply:
That test is coming on another ticket, but this was a quick and dirty way to make sure we don't completely regress the SubCode code.